tests/devices/i04/test_murko_results.py

```diff
@@ -807,3 +810,23 @@ async def test_trigger_stops_if_no_message_in_5_seconds_and_raises_warning(
         and record.levelname == "WARNING"
         for record in caplog.records
     )
+
+
+async def test_if_redis_connection_failed_then_no_error_is_raised_and_motors_set_to_0(
+    murko_results: MurkoResultsDevice,
+):
+    murko_results.redis_client.ping = AsyncMock(side_effect=ConnectionError)
+    await murko_results.stage()
+    await murko_results.trigger()
+    await murko_results.unstage()
+    assert await murko_results.x_mm.get_value() == 0
```

Comment:
Should: See comment above. Either this test or another should confirm that if we set some rubbish values into the x/y/z before stage it will reset them